### PR TITLE
feature: added routes for views

### PR DESCRIPTION
**Pull request by SukeruB**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,5 +24,8 @@
   "dependencies": {
     "express": "^4.17.1",
     "pg": "^8.7.1"
+  },
+  "devDependencies": {
+    "dotenv": "^10.0.0"
```
Comment (pabloverduzco) [inline package.json:29]:
```suggestion
    "dotenv": "^10.0.0"
    "nodemon": "^2.0.12"
```

File: src/database-connection.js [hunk 2]
```diff
@@ -1,13 +1,16 @@
 const {Router} = require('express');
 const {Pool} = require('pg');
 const router = Router();
+const dotenv = require('dotenv');
+
+const {HOST,USER,PASSWORD,DATABASE,PORT} = process.env;
```
Comment (pabloverduzco) [inline src/database-connection.js:6]: https://github.com/mysticatea/eslint-plugin-node/blob/master/docs/rules/no-process-env.md

**Review by pabloverduzco — CHANGES_REQUESTED** [review]
-

File: src/database-connection.js [hunk 3]
```diff
@@ -2,8 +2,14 @@ const {Router} = require('express');
...
 const dotenv = require('dotenv');
+const config = require("./config")
 
-const {HOST,USER,PASSWORD,DATABASE,PORT} = process.env;
+
+//const {HOST,USER,PASSWORD,DATABASE,PORT} = process.env;
+
+if (config.env === "development"){
+    HOST,USER,PASSWORD,DATABASE,PORT
+}
```
Comment (pabloverduzco) [inline src/database-connection.js:12]: 🤔

File: src/database-connection.js [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
-    user: 'postgres',
-    password: '',
-    database: '',
-    port: '5432'
+    host: HOST,
+    user: USER,
+    password: PASSWORD,
+    database: DATABASE,
+    port: PORT
 });
```
Comment (SukeruB) [inline src/database-connection.js:11]: 🍆🍆🍆

**Review by SukeruB — COMMENTED** [review]
🍆🍆

**Pull request merged**